web/instances/instances.go

```diff
@@ -85,6 +84,17 @@ func createHandler(c echo.Context) error {
 			return wrapError(err)
 		}
 	}
+	if iterations := c.QueryParam("KdfIterations"); iterations != "" {
+		iter, err := strconv.ParseInt(iterations, 10, 64)
+		if err != nil {
+			return wrapError(err)
+		}
+		if iter < 5000 && iter != 0 {
+			err := errors.New("The KdfIterations number is too low")
+			return jsonapi.InvalidParameter("KdfIterations", err)
+		}
+		opts.KdfIterations = int(iter)
```

Comment:
Pourquoi cette conversion sachant que `iter` est un `int64` ? D'après ce que je peux lire dans la godoc, `int` n'a pas la même taille selon les implémentations et plateformes. N'est-ce pas aussi risqué de ce fait ?

Reply:
Ça ne pose pas de problème, même sur 32 bits, le nombre d'itérations reste bien inférieur à la limite.